### PR TITLE
footer redesign to include voter's edge, register to vote, OO & PEC credit line

### PR DESCRIPTION
**Pull request by ckingbailey**

### CODE REVIEW

**Comment by ckingbailey** [comment]
I folded in register to vote link, credit line update, and the line at the bottom stipulating where our data comes from per [@elinaru redesign](https://github.com/caciviclab/odca-jekyll/issues/199#issuecomment-418586186) #199

**Comment by ckingbailey** [comment]
screen shot 2018-09-09 at 9 41 06 pm
desktop

screen shot 2018-09-09 at 9 41 45 pm
mobile

**Comment by ckingbailey** [comment]
#199 #200

File: _includes/footer.html [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+          </h5>
+          <h6 class="footer__stipulation">All contribution and expenditure data is provided directly by city officials for their respective city ballots</h6>
+        </div>
       </div>
-
-      <div class="footer__col grid-col-6">
-        <span class="tagline">Brought to you by <span class="tagline__em">California Civic Lab</span></span>
+      <div class="footer__col grid-col-4">
+        <div class="footer__voters-edge-widget">
+          <iframe title='Enter your address for personalized voting information' src='https://votersedge.org/en/address-widget' frameBorder='0' scrolling='no' style='width:240px;height:198px;'></iframe>
```
Comment (adborden) [inline _includes/footer.html]: I think the height could be bumped up to 200px. I'm seeing the bottom border get cut off.
screenshot from 2018-09-09 22-45-28

File: _sass/module/_footer.scss [hunk 2]
```diff
@@ -4,6 +4,10 @@
   background-color: $color-grey-5;
   padding-bottom: $spacing-base * 2;
 
+  a {
+    text-decoration: none;
```
Comment (adborden) [inline _sass/module/_footer.scss]: @elinaru should this be applied site-wide? Should all of our `<a>` tags have no decoration?

**Comment by adborden** [comment]
I'm going to fix the conflict and get this merged.

**Pull request merged**